gameboard/game.scss

```diff
@@ -0,0 +1,197 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+$base-font-color: #fff;
+$btn-color: #243c60;
+$bgc-color: rgba(0, 0, 0, 0.45);
+$btn-color-hover: #051e42;
+$base-font-family: "Lato", sans-serif;
+$bg-url: url("./bg.png");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: $base-font-color;
+  font-family: $base-font-family;
+}
+
+.wrapper {
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  background-image: $bg-url;
+
+  &__main {
+    background-color: $bgc-color;
+    height: 100%;
+    width: 90%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+    position: relative;
+  }
+}
+
+.timer {
+  position: absolute;
+  top: 5%;
+  &__counter {
+    font-size: 48px;
+    font-weight: bold;
+  }
+}
+.gameBoard {
+  &__wrapper {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+  }
+
+  &__player {
+    position: absolute;
+    top: 10%;
+
+    &-name {
+      font-size: 40px;
+    }
+  }
+
+  &__board {
+    width: 400px;
+    height: 400px;
+    background-color: #fff;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+
+    &-fieldsWrapper {
```

Comment:
If you were thinking about using BEM then the `fieldsWrapper` is not a good usage of BEM. Besides, BEM is a CSS writing style that is rather an enemy of nestings especially, when it is possible. If you name things connecting a few names in most cases it means:
- not good class granulation,
- wrong naming convention for single elements,
- wrong template construction and nesting.
BEM supports nesting only when it is necessary.